src/store/reducers/data.js

```diff
@@ -52,6 +58,8 @@ const dataAndOffersRecapReducer = (state = initialState, action) => {
       return { ...state, ...{ venues: action.venues } }
     case SET_VENUE_PROVIDERS:
       return { ...state, ...{ venueProviders: [action.payload] } }
+    case SET_USERS:
+      return { ...state, ...{ users: action.users } }
```

Comment:
on pourrait avoir ici : 

```suggestion
  case UPDATE_USER:
      if (!users || !users.length) return state
      const newUser = {state.users[0],  ...action.user }
      return {...state, }
      return { ...state, ...{ users: [newUser] } }
```

Ainsi, on à pas besoin de récupéré currentUser dans note nouvelle home. Avec : 
```javascript
dispatch(updateUsers({ hasSeenProTutorials: true }))
```

C'est juste une idée.